picasso/src/main/java/com/squareup/picasso3/Request.java

```diff
@@ -434,18 +435,14 @@ public Builder purgeable() {
 
     /** Decode the image using the specified config. */
     public Builder config(@NonNull Bitmap.Config config) {
-      if (config == null) {
-        throw new IllegalArgumentException("config == null");
-      }
+      checkNotNull(config, "config == null");
```

Comment:
any concerns about costs of string allocation/gc during request creation during, say, a list flinging?

Reply:
It's in the constant pool so it'll only be a `ldc` bytecode. I'm happy to inline these in a follow-up though.